Automattic/wp-calypso
# Premium themes: Add correct cart item for Atomic sites

Fixes https://github.com/Automattic/wp-calypso/issues/65285

#### Proposed Changes

Ensures that the correct product is added to the cart when purchasing a premium theme on an Atomic site with a lower-tier plan.

#### Testing Instructions

- Use the Calypso live link below.
- Go to `/theme/alonso`.
- Select an Atomic site with a lower-tier plan (e.g. Starter plan).
- Click on "Pick this design".
- Make sure the theme is added to the cart correctly and that you can proceed with the purchase.
- Go to `/theme/alonso` again.
- Select a Jetpack site with a Free plan.
- Click on "Pick this design".
- Make sure the Jetpack Complete plan is added to the cart.
